packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -371,7 +372,8 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
  * @remarks TODO: examples of suggested actions for recovery.
  * - How would a user delete the created document?
  * - What would a retry pattern look like here?
- * @internal
+ * @legacy
+ * @alpha
```

Comment:
If possible (if users don't need to subclass this) this, and all other classes exposed in this change should be marked `@sealed`, so that adding and renaming private and protected members is known to be non breaking.

Reply:
that would technically be a breaking change, since it wasn't previously sealed.

Reply:
it could be sealed in 2.20 if someone wants to take the effort to track it, do the notification, and make the change

Reply:
We would not have to categorize this as a breaking change as it was not documented as supported. Contractually and practically, there is no reason it couldn't be sealed. The change was for Loop usage and there is no derivation.
Since the only use is for testing in Loop we also have opportunity to surface narrow helper functionality to cover just their case and not even make this `@legacy`. Then `DocumentPostCreateError` and `IRouterliciousDriverPolicies` also aren't stuck in `@legacy`.